tfjs-backend-webgpu/src/binary_op_util.ts

```diff
@@ -201,26 +178,54 @@ const PRELU_VEC4 = `
 const SQUARED_DIFFERENCE = 'return (a - b) * (a - b);';
 const SUB = 'return a - b;';
 
-function getBinaryWithNanString(op: string, useVec4: boolean) {
-  const checkNanSnippet = useVec4 ? CHECK_NAN_SNIPPET_VEC4 : CHECK_NAN_SNIPPET;
-  return useVec4 ? `
-    var resultTemp = vec4<f32>(${op}(a, b));
-    ` + checkNanSnippet +
-          `
-    return resultTemp;
-  ` :
-                   checkNanSnippet + `
-    return ${op}(a, b);
-  `;
-}
+const ATAN2 = 'var resultTemp = atan2(a, b);';
+const MAX = 'var resultTemp = max(a, b);';
+const MIN = 'var resultTemp = min(a, b);';
 
 export function getBinaryOpString(
     type: BinaryOpType, useVec4?: boolean): string {
+  // NaN-sensitive ops
```

Comment:
Perhaps "sensitive"/"insensitive" is not the right word here, but I'm not sure what could be better.

Reply:
Maybe "Ops with NaN check" and "Ops without NaN check"?